playwright.config.ts

```diff
@@ -0,0 +1,81 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
```

Comment:
consider removing

Reply:
I'd kinda rather keep it for now since this is early in the usage of Playwright and it might come in handy later. Happy for an engineer to remove/cleanup later themselves if they don't feel they will need it.